docs/markdown/snowplow_media_player_overview.md

```diff
@@ -0,0 +1,287 @@
+
+### 2. Configuring the web model (in case it has not been run before)
+
+Please refer to the `Quick Start` guide within the [snowplow-web dbt doc site][dbt-snowplow-web] to make sure you configure the web model appropriately. (e.g.: checking the source data or enabling desired contexts).
+
+One thing to highlight here: as the package is built onto the snowplow_incremental_materialization logic provided by the web package, please leave the `snowplow__incremental_materialization` variable as is with the default `snowplow_incremental` value.
+
+### 3. Adding the selector.yml file
+
+Within this package we have provided a suite of suggested selectors to run and test the models within the package together with the web model. This leverages dbt's [selector flag][dbt-selectors].
+
+The selectors include:
+
+- `snowplow_web`: Recommended way to run the package. This selection includes all models within the Snowplow Web as well as any custom models you have created. This is the same as in the web package
+- `snowplow_web_lean_and_media_player_tests`: Recommended way to test the models within the web and media player packages. See the testing section for more details. There are other selectors for testing, please see the Tests section for more details on this.
+
+These are defined in the `selectors.yml` file within the package, however in order to use these selections you will need to copy this file into your own dbt project directory. This is a top-level file and therefore should sit alongside your `dbt_project.yml` file.
+
+## Operation
+
+Due to its unique relationship with the web package, in order to operate the media player package together with the web model there are several considerations to keep in mind. Depending on the use case one of the following scenarios may happen:
+
+1. The web package is already being used and the media tracking package needs to be added at a later time.
+2. The web package has not been used but it needs to be run together with the media player package.
+3. Only the media player package needs to be run.
+------
+### 1. Adding media player data model to an existing dbt project with web model data already running
+
+Supposing there are months of data being collected using the web package and media tracking is introduced at that later stage there is no need to fully reprocess the web data from the date media tracking was deployed.
+
+As models from both packages need to be run in synch, first the backfilling of the models from the new package needs to happen. Please note that during backfill no new web data is allowed to be processed and depending on the `snowplow_backfill_limit_days` configured and the period that needs backfilling it can take a while for all models to synch up and new web events to be processed.
+
+To begin the synching process please run the following script:
+
+```bash
+dbt run -m snowplow_web.base snowplow_media_player --vars 'snowplow__start_date: date_when_media_player_tracking_started'
```

Comment:
You mean this way? dbt run -m snowplow_web.base snowplow_media_player --vars 'snowplow__start_date: <<YOUR_FIELD_HERE>>' The variable needs to be this one in this case.

Reply:
Actually the convention is like this `<target.schema>`, so I guess it would be like this:
dbt run -m snowplow_web.base snowplow_media_player --vars 'snowplow__start_date: <date_when_media_player_tracking_starts>'

Reply:
I like this more, explanatory like how it was before plus making sure the user catches the value to overwrite due to the  < >